operations/customers.md

```diff
@@ -0,0 +1,169 @@
+<!-- AUTOMATICALLY GENERATED, DO NOT MODIFY -->
+# Customers
+
+## Get customers
+
+This operation returns a list of customers
+
+**Note:** This operation needs [Authentication](../guidelines/authentication.md) and supports the following JSON:API features:
+
+- [Sparse fieldsets](../guidelines/sparse-fieldsets.md) - supports all fields of `customer` query parameter.
+
+### Request
+
+`GET` `[PlatformAddress]/api/v2/customers`
+
+### Response
+
+```javascript
+{
+  "data": [
+    {
+      "id": "31b14937-2524-491f-b0a0-dc0a7393ff33",
+      "type": "customers",
+      "attributes": {
+        "email": "example@email.com",
+        "fullName": "Name",
+        "taxNumber": "tax1234",
+        "dateOfBirth": "2024-10-24",
+        "phone": "+123123123",
+        "mobile": "+123123123",
+        "address1": "Address",
+        "address2": "Address",
+        "city": "City",
+        "state": "State",
+        "postalCode": "10000",
+        "country": "HR",
+        "notes": "This is a note",
+        "countrySpecificCode": "code",
+        "companyName": "Company name",
+        "createdAt": "2024-10-16T14:30:00Z",
+        "updatedAt": "2024-10-19T14:30:00Z"
+      }
+    }
+  ],
+  "links": {
+    "prev": "https://pos.mews-demo.com/api/v2/customers?page%5Bbefore%5D=NA&page%5Bsize%5D=1",
+    "next": "https://pos.mews-demo.com/api/v2/customers?page%5Bafter%5D=NA&page%5Bsize%5D=1"
+  }
+}
+```
+Below is a list of all possible fields this endpoint can return including relationships fields fetched with include query parameter.
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `fullName` | string,null | optional, max length 255 characters | The full name of the customer. This can be either a personal name or a company name. |
```

Comment:
The docs would be easier to read if the fields in the JSON examples are in the same order as they are listed in the property table, can we align them please?